pkg/statistics/handle/handle.go

```diff
@@ -171,40 +171,36 @@ func NewHandle(
 	return handle, nil
 }
 
-// GetTableStats retrieves the statistics table from cache, and the cache will be updated by a goroutine.
-// TODO: remove GetTableStats later on.
-func (h *Handle) GetTableStats(tblInfo *model.TableInfo) *statistics.Table {
```

Comment:
What's the problem with the old API?
Since it changes so many lines and and it make me confused that, is it an unrelated refactor?